themes/custom/templates/index.html

```diff
@@ -1,25 +1,20 @@
 {% import "macros.html" as macros %}
-<!doctype html>
+<!DOCTYPE html>
 <html class="dark" lang="{{ lang | default(value='en') }}">
```

Comment:
class="dark" is probably not needed.

```suggestion
<html lang="{{ lang | default(value='en') }}">
```